dev-python/rdflib/rdflib-4.2.2.recipe

```diff
@@ -0,0 +1,109 @@
+SUMMARY="Python library for working with RDF"
+DESCRIPTION="RDFLib is a Python library for working with RDF, a simple \
+yet powerful language for representing information as graphs."
+HOMEPAGE="https://github.com/RDFLib/rdflib"
+COPYRIGHT="2002-2017 RDFLib Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/RDFLib/rdflib/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="a362926f77624a8abfb93042c28607f49445d003792d5271cd07f0610ab16033"
+SOURCE_FILENAME="rdflib-$portVersion.tar.gz"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	rdflib = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python python3)
+PYTHON_VERSIONS=(2.7 3.6)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_$pythonPackage=\"
+		${portName}_$pythonPackage = $portVersion
+		cmd:csv2rdf_$pythonVersion
+		cmd:rdf2dot_$pythonVersion
+		cmd:rdfgraphisomorphism_$pythonVersion
+		cmd:rdfpipe_$pythonVersion
+		cmd:rdfs2dot_$pythonVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		cmd:python$pythonVersion
+		\""
+	if [ $pythonPackage = python3 ]; then
+		PROVIDES_python3+="
+			cmd:csv2rdf
+			cmd:rdf2dot
+			cmd:rdfgraphisomorphism
+			cmd:rdfpipe
+			cmd:rdfs2dot
+			"
+	fi
+	BUILD_REQUIRES="$BUILD_REQUIRES
+		setuptools_$pythonPackage"
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+		cmd:python$pythonVersion"
+done
+
+BUILD()
+{
+	for i in "${!PYTHON_PACKAGES[@]}"; do
+		pythonPackage=${PYTHON_PACKAGES[i]}
+		pythonVersion=${PYTHON_VERSIONS[$i]}
+
+		rm -rf "$sourceDir"-$pythonPackage
+		cp -a "$sourceDir" "$sourceDir"-$pythonPackage
+		cd "$sourceDir"-$pythonPackage
+
+		python=python$pythonVersion
+		$python setup.py build
+	done
+}
+
+INSTALL()
+{
+	for i in "${!PYTHON_PACKAGES[@]}"; do
+		pythonPackage=${PYTHON_PACKAGES[i]}
+		pythonVersion=${PYTHON_VERSIONS[$i]}
+
+		cd "$sourceDir"-$pythonPackage
+
+		python=python$pythonVersion
+		installLocation=$prefix/lib/$python/vendor-packages/
+		export PYTHONPATH=$installLocation
+		mkdir -p "$installLocation"
+		$python setup.py install \
+			--root=/ --prefix="$prefix"
+
+		if [ "$pythonPackage" = python ]; then
+			rm -rf "$prefix"/bin
```

Comment:
```suggestion
			rm -rf "$binDir"
```

Since this is an `any` architecture package, `$binDir` and `$prefix/bin` is the same.